### DISCUSSION TITLE
How to use autodiscoverFilter

### DISCUSSION THREAD

**Discussion by rubyisbeautiful** [renovatebot/renovate#25240] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

6.2.0

### If you're self-hosting Renovate, select which platform you are using.

github.com

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

Use autodiscover to filter from all repositories the GH token gives access to

#### use autodiscoverFilter with repo name minimatch patterns

`autodiscoverFilter: ["foo-*"]`

#### use autodiscover with filename-in-repo patterns

`autodiscoverFilter: ["/.enable-this-repo"]`

It isn't clear from the [autodiscover docs](https://docs.renovatebot.com/self-hosted-configuration/#autodiscoverfilter) what is happening.

I get "DEBUG: None of the discovered repositories matched the filter" with both of the above.  With the latter, I added a file to one of the repos that the GH app has access to.

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by rarkins** [comment, marked as answer]
It filters full repo name (org-name/repo-name) because a token could have access to multiple orgs. Check the example in the docs

**Reply by rubyisbeautiful** [reply]
Thanks @rarkins but I don't think the documentation is explaining well.  There are no examples that show the pattern you gave [here](https://docs.renovatebot.com/self-hosted-configuration/#autodiscoverfilter).  This particular example `"/myapp/{readme.md,src/**}"` is what led me to think perhaps it would look for a file.
The minimatch example, `{  "autodiscoverFilter": ["project/*"] }`, is ambiguous.  

I'll try the org-name/repo-name.  I could also do a PR for docs based on my understanding and the results.

**Reply by rarkins** [reply]
I think project/* example is directly relevant but agree with you on the env example with file names, which is confusing. PR welcome!

**Reply by rubyisbeautiful** [reply]
My attempt to clarify via [PR](https://github.com/renovatebot/renovate/pull/25242).  Thanks!
